receiver/ciscoosreceiver/metadata.yaml

```diff
@@ -0,0 +1,41 @@
+type: ciscoosreceiver
+
+status:
+  class: receiver
+  stability:
+    development: [metrics]
+  distributions: []
+  codeowners:
+    active: [dmitryax]
+
+resource_attributes:
+
+attributes:
+  target:
+    description: The target Cisco device hostname or IP address
+    type: string
+    enabled: true
+
+metrics:
+  cisco_up:
```

Comment:
We do not typically use `_` in metric names. We prefer periods.

Using `cisco` as a metric namespace might also be somewhat difficult.

If we are targeting connectivity, can we use `cisco.device.connected` instead?

Reply:
also, and just food for thought. Typically you'd want cumulative counters that count the number of seconds the connection is active, so you can see if there was a disconnect in between reporting periods.

So `cisco.device.connection.age` with a unit of second.